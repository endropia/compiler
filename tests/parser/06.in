const
a = 1;
const
b : Integer = 2;
c = '1';
d = 3 + 7;
begin
end.

