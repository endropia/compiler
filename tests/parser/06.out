program : Unnamed program
   const:
      a
      1
   const:
      b
      type: Integer
      2
   const:
      c
      1
   const:
      d
      +
         3
         7
   stmts:
      empty